.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,24 @@
+name: Docker Compose CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test-compose:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Private Registry
+        uses: docker/login-action@v1
+        with:
+          registry: docker.wso2.com
+          username: ${{ secrets.WSO2_SUB_USER }}
+          password: ${{ secrets.WSO2_SUB_PASS }}
```

Comment:
these secrets will need to be added or I can change the keys to use existing secrets